libbfio/src/handle.rs

```diff
@@ -201,53 +255,51 @@ extern "C" {
     ) -> c_int;
 }
 
-impl Drop for Handle {
-    fn drop(&mut self) {
-        use libyal_rs_common::ffi::AsTypeRef;
-        use log::trace;
-
-        let mut error = ptr::null_mut();
-
-        trace!("Calling `{}`", module_path!());
-
-        unsafe {
-            libbfio_handle_free(&mut self.as_type_ref_mut() as *mut _, &mut error);
-        }
-
-        debug_assert!(error.is_null(), "`{}` failed!", module_path!());
-    }
-}
-
 impl Handle {
```

Comment:
Document what this is? (aka vs IoHandle)

Reply:
I added some module-level docs